community/technical-team.md

```diff
@@ -4,16 +4,18 @@ title: "Technical team"
 ---
 # Technical team
 
-The Technical Team consists of interested developers and two Board-appointed Co-Chairs. The Co-Chairs act as administrative chairs of the Technical Team, e.g. calling meetings and coordinating development efforts. They also have administrative privileges for the MEI repositories. The MEI Technical Team holds regular virtual and in-person meetings, maintains the MEI repository on GitHub ([https://github.com/music-encoding](https://github.com/music-encoding)), prepares new releases of MEI by reviewing and implementing revisions and features requested by the community, and supports the wider MEI community with their technical expertise.
+The Technical Team consists of interested developers and two Board-appointed Co-Chairs. The Co-Chairs act as administrative chairs of the Technical Team, e.g., calling meetings and coordinating development efforts. They also have administrative privileges for the MEI repositories. The MEI Technical Team holds regular virtual and in-person meetings, maintains the MEI repository on GitHub ([https://github.com/music-encoding](https://github.com/music-encoding)), prepares new releases of MEI by reviewing and implementing revisions and features requested by the community, and supports the wider MEI community with their technical expertise.
 
 * **Benjamin W. Bohl (Technical Co-Chair)**, Institut für Musikwissenschaft, Goethe Universität Frankfurt, Frankfurt am Main and Musikwissenschaftliches Seminar Detmold/Paderborn
-* **Johannes Kepper (Technical Co-Chair)**, Musikwissenschaftliches Seminar Detmold/Paderborn
-* **Andrew Hankinson**, RISM Digital Center, Bern
-* **Perry Roland**, Alderman Library, University of Virginia, Charlottesville
```

Comment:
Hm. This is a pretty big step; I feel @pe-ro has earned a permanent spot on the Technical Team. So I'd like to see a bit more of a discussion about this move beyond a simple PR. Was this discussed at any point previously?

Reply:
I agree. There has been some discussion (at least between the TT Co-chars) but no final decision)

Reply:
Without Perry, there would be no such thing as a technical team, so imho it is imperative to have him on that list.